editoast, front: simplify stdcm failure case

Erashin opened the pull request:
When stdcm fails, remove complex and non-useful code returning conflicts along the shortest path. Return simplified string: `Incompatibilité avec des sillons et/ou planches travaux existants.`  /  `Clash with existing train and/or work schedules.` -> validated by @Arthur-Lefebvre.

The following image has the older version of the message, but depicts the general nature of the change on the front side (PR has the lastest version of the message):
<img width="733" height="293" alt="image" src="https://github.com/user-attachments/assets/4375eed6-15f7-4784-943c-42928dbf8161" />

Erashin committed:
editoast: simplify stdcm failure case

When stdcm fails, remove complex and non-useful code returning conflicts along the shortest path. Return empty failure error.

Signed-off-by: Erashin <alwenn.charpentier@gmail.com>

flomonster reviewed (approved):
Editoast part LGTM.
So simple now!

Erashin committed:
front: simplify stdcm failure case

When stdcm fails, remove complex and non-useful code returning conflicts along the shortest path. Return simple string instead.

Signed-off-by: Erashin <alwenn.charpentier@gmail.com>

eckter reviewed (approved):
LGTM for tests (yeah that was easy)

Math-R reviewed (approved):
lgtm, thanks much more simple like this !
